classroom.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'student'
+
+class Classroom
+  attr_accessor :lable, :students
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement `:students` accessor to `attr_reader` 🙏🏼 

```ruby
attr_reader :students
```